README.md

```diff
@@ -1111,6 +1111,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Thanga Mariappan](https://github.com/thangamariappan19)
   - [Tharaka Bandara](https://github.com/Ttecs)
   - [Thisaru Dilshan](https://github.com/ThisaruD)
+  - [Thirumalesh Chinna](https://github.com/imtmh)
```

Comment:
Please add your name above `Thisaru Dilshan` (line 1113)

Reply:
Thanks, I have just updated it.